metsrw/metadata.py

```diff
@@ -125,14 +131,14 @@ def get_status(self):
 
         :returns: None or the STATUS string.
         """
-        if self.status:
+        if self.status is not None:
             return self.status
         if self.subsection == 'dmdSec':
             if self.older is None:
                 return 'original'
             else:
                 return 'updated'
-        if self.subsection == 'rightsMD':
+        if self.subsection in ('techMD', 'rightsMD',):
             # TODO how to handle ones where newer has been deleted?
             if self.newer == None:
```

Comment:
Is there a reasons you're using `== None` instead of `is None`? (I don't see a custom `__eq__` on this class...)